packages/graphback-core/tests/__snapshots__/EngineTests.ts.snap

```diff
@@ -0,0 +1,21 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Sadly Snapshots in Jest are much less readable :(. Nothing to do here anyway

Reply:
I can try see if I can enable the markdown too.